src/tests/ftest/scrubber/performance.yaml

```diff
@@ -0,0 +1,64 @@
+hosts:
+  test_servers: 2
+  test_clients: 2
+timeout: 1800
```

Comment:
Looks like the test took under 4 minutes here
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13268/9/testReport/FTEST_scrubber/
Can we reduce this timeout?

Reply:
Done. Reduced to 660.